ops/testing.py

```diff
@@ -319,16 +319,33 @@ def set_leader(self, is_leader=True):
         if is_leader and not was_leader and self._charm is not None and self._hooks_enabled:
             self._charm.on.leader_elected.emit()
 
+    def get_backend_calls(self, reset=False):
```

Comment:
While here, I went ahead and changed the default to "reset=True'. Having written a few tests, it is usually quite a bit clearer to just list the calls since the last time you checked, rather than having a growing list that says [('a',)] then [('a',),('b',)], then ([('a',), ('b',), ('c'),)]

best to change it before anyone depends on it.